src/cromshell/metadata/command.py

```diff
@@ -0,0 +1,141 @@
+import logging
+
+import click
+import requests
+
+from cromshell.utilities import cromshellconfig, http_utils, io_utils
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command(name="metadata")
+@click.argument("workflow_id")
+@click.option(
```

Comment:
Using clicks multiple options parameter is the easiest way to pass a list of keys from the cli, but it requires that users add `-k` before every key. (e.g. -k id -k status ...). There is a way to have it accept a list instead (e.g. -k [id, status]) but that requires custom code so I left it out but its mentioned [here](https://stackoverflow.com/questions/47631914/how-to-pass-several-list-of-arguments-to-click-option) if we find it valuable.